pipelines/powershell-package/default-ci-pipeline.yaml

```diff
@@ -0,0 +1,41 @@
+parameters:
+  excludeRules: [] # List of rules. See https://github.com/PowerShell/PSScriptAnalyzer/tree/master/RuleDocumentation
+  nugetversion: '5.4.0'
+  pathToProjects: '$(Build.SourcesDirectory)/src/'
+  nugetDevOpsFeedToPublish: '/8187f5c9-e9c1-419f-8a5c-98285cf7633c' #GlobalDev
+  pushpackage: 'false'
+  noUnitTests: 'false'
+  vmImage: 'ubuntu-latest'
+  jobName: 'Build'
+
+jobs:
+- job: '${{ parameters.jobName }}'
+  
+  pool:
+    vmImage: '${{ parameters.vmImage }}'
+
+  steps:
+  - template: templates/prerequisities.yaml
+    parameters:
+      nugetversion: '${{ parameters.nugetversion }}'
+
+  - template: ../shared-templates/gitversion.yaml
+
+  - template: templates/register.yaml
+    parameters:
+      pathToProjects: '${{ parameters.pathToProjects }}'
+
+  - template: templates/test.yaml
+    parameters:
+      pathToProjects: '${{ parameters.pathToProjects }}'
+      excludeRules: '${{ parameters.excludeRules }}'
+      noUnitTests: '${{ parameters.noUnitTests }}'
+
+  - template: templates/pack.yaml
+    parameters:
+      pathToProjects: '${{ parameters.pathToProjects }}'
+
+  - template: templates/push.yaml
+    parameters:
+      nugetDevOpsFeedToPublish: '${{ parameters.nugetDevOpsFeedToPublish }}'
+      pushpackage: '${{ parameters.pushpackage }}'
```

Comment:
the same here (if you want the pushpackage to be set during runtime it can't be passed like this AFAIK)